### PR TITLE
Use consistent tracking code format

### LINKED ISSUE #82: Minimize the length of the tracker
The tracker generates words per byte and this can lead to a quite long phrase. It would perhaps be more helpful to abbreviate this into a shorter piece by using another tool other than bytes.

### PR DESCRIPTION
**Pull request by AddressXception**
### Issue
*Link your PR to an issue*

Fixes #82 

### Description
Follow Josh's algorithm for the tracking code

### Testing
`make test`

### Checklist
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] 🤔 **CONSIDER** adding a unit test if your PR resolves an issue.
- [ ] ✅ **DO** check open PR's to avoid duplicates.
- [ ] ✅ **DO** keep pull requests small so they can be easily reviewed.
- [ ] ✅ **DO** build locally before pushing.
- [ ] ✅ **DO** make sure tests pass.
- [ ] ✅ **DO** make sure any new changes are documented.
- [ ] ✅ **DO** make sure not to introduce any compiler warnings.
- [ ] ❌**AVOID** breaking the continuous integration build.
- [ ] ❌**AVOID** making significant changes to the overall architecture.

💚Thank you!

### CODE REVIEW

File: src/electionguard/tracker.py [hunk 1]
```diff
@@ -39,32 +39,29 @@ def tracker_hash_to_words(
...
+        # Select 4 bytes for the segment
+        first = segments[i]
+        second = segments[i + 1]
+        third = segments[i + 2]
+        fourth = segments[i + 3]
 
+        # word is byte(1) + 1/2 of byte(2)
+        word_part = get_word((first * 16 + (second >> 4)))
 
-def tracker_words_to_hash(
```
Comment (keithrfung) [inline src/electionguard/tracker.py:53]: The `electionguard-web-api` currently exposes this. Is there no case where a dev may search by the tracker_code but lookup by tracker_hash? Or do we want to add a ticket for someone to create this function for `hacktoberfest`

CC @rkorsak

**Review by keithrfung — CHANGES_REQUESTED** [review]
Lint failing. 

Respond to comments.

File: src/electionguard/tracker.py (hunk 1 above)
Comment (AddressXception) [reply src/electionguard/tracker.py:53]: It's not used anywhere in the protocol and specifically it is not used in generating any cryptographic values.  i think it makes sense to discourage using the tracking code anywhere except as a human-readable lookup value in the published results.  further, this representation is entirely arbitrary.  My vote is to remove the endpoint from the web api

Comment (keithrfung) [reply src/electionguard/tracker.py:53]: Fair point. Just denoting it.

**Review by keithrfung — APPROVED** [review]

**Pull request merged**